### ISSUE TITLE
TypeError: invalid 'in' operand Radio

### ISSUE THREAD

**Issue by numerogeek** [avanzu/AdminThemeBundle#141]
Hi guys,

I have updated to the latest dev-master, runned asset-install & avanzu-fetch-vendors without any error; but in the front I have a JS error that block everything else: 

> TypeError: invalid 'in' operand Radio
> ...ackbone;_="default"in _?_["default"]:_;Radio="default"in Radio?Radio["default"]:...
> in admin-lte-all.js line 24 col 796

Have you ever heard about this one ?

EDIT: This comes from the last version of MarionnetteJS.
To reproduce the problem, you can make a fresh install, fetch the assets & build them.

**Comment by numerogeek** [comment]
I think that #140 will fix it.

**Comment by Notmarrco** [comment]
fwiw it works with ``` "marionette"                : "~2.4.4", ``` while the PR is not merged.

**Comment by jewome62** [comment]
Hello, 

I have the same error : 

`TypeError: invalid 'in' operand Radio  admin-lte-all.js:17:3780`

```
root@panel:/var/www/panel# composer show avanzu/admin-theme-bundle
Do not run Composer as root/super user! See https://getcomposer.org/root for details
name     : avanzu/admin-theme-bundle
descrip. : Admin Theme based on the AdminLTE Template for easy integration into symfony
keywords : 
versions : * dev-master
type     : library
license  : MIT License (MIT) (OSI approved) https://spdx.org/licenses/MIT.html#licenseText
source   : [git] https://github.com/avanzu/AdminThemeBundle.git c0ff9dd38626778e3a7ac757a837677cb21e0e0c
dist     : [zip] https://api.github.com/repos/avanzu/AdminThemeBundle/zipball/c0ff9dd38626778e3a7ac757a837677cb21e0e0c c0ff9dd38626778e3a7ac757a837677cb21e0e0c
names    : avanzu/admin-theme-bundle

autoload
psr-4
Avanzu\AdminThemeBundle\ => .

requires
php >=5.3.0
symfony/assetic-bundle >=2.3
symfony/event-dispatcher >=2.3
symfony/http-foundation >=2.3
symfony/http-kernel >=2.3
symfony/twig-bundle >=2.3

requires (dev)
hamcrest/hamcrest-php dev-master
mockery/mockery 0.9.*@dev
```

I make regenerate all vendor and build assets without success.
Do you have a solution ?

Thanks

**Comment by shakaran** [comment]
@jewome62 @Notmarrco @numerogeek The issue should be now fixed since #140 has been merged. Please test master branch and reopen if you still have issues. Closing as fixed. Thanks for your contributions and comments!
